When testing `features`, the following error occurs:

```
PS G:\sail> ./vcpkg install sail[*]

...

    sail[all,avif,bmp,core,gif,high-priority-codecs,highest-priority-codecs,ico,jpeg,jpeg2000,jpegxl,low-priority-codecs,lowest-priority-codecs,medium-priority-codecs,openmp,pcx,png,psd,qoi,svg,tga,tiff,wal,webp,xbm]:x64-windows@0.9.3

...

Installing 23/23 sail[all,avif,bmp,core,gif,high-priority-codecs,highest-priority-codecs,ico,jpeg,jpeg2000,jpegxl,low-priority-codecs,lowest-priority-codecs,medium-priority-codecs,openmp,pcx,png,psd,qoi,svg,tga,tiff,wal,webp,xbm]:x64-windows@0.9.3...
Building sail[all,avif,bmp,core,gif,high-priority-codecs,highest-priority-codecs,ico,jpeg,jpeg2000,jpegxl,low-priority-codecs,lowest-priority-codecs,medium-priority-codecs,openmp,pcx,png,psd,qoi,svg,tga,tiff,wal,webp,xbm]:x64-windows@0.9.3...
-- Downloading https://github.com/HappySeaFox/sail/archive/v0.9.3.tar.gz -> HappySeaFox-sail-v0.9.3.tar.gz...
-- Extracting source G:/sail/downloads/HappySeaFox-sail-v0.9.3.tar.gz
-- Applying patch fix-include-directory.patch
-- Using source at G:/sail/buildtrees/sail/src/v0.9.3-596dc8266e.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: "C:/Program Files/Microsoft Visual Studio/2022/Enterprise/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v
    Working Directory: G:/sail/buildtrees/sail/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      G:\sail\buildtrees\sail\config-x64-windows-dbg-CMakeCache.txt.log
      G:\sail\buildtrees\sail\config-x64-windows-rel-CMakeCache.txt.log
      G:\sail\buildtrees\sail\config-x64-windows-out.log

Call Stack (most recent call first):
  installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:252 (vcpkg_execute_required_process)
  ports/sail/portfile.cmake:27 (vcpkg_cmake_configure)
  scripts/ports.cmake:175 (include)
```
**G:\sail\buildtrees\sail\config-x64-windows-out.log:**
```
-- Found ZLIB: optimized;G:/sail/installed/x64-windows/lib/zlib.lib;debug;G:/sail/installed/x64-windows/debug/lib/zlibd.lib (found version "1.3.1")
-- Found PNG: optimized;G:/sail/installed/x64-windows/lib/libpng16.lib;debug;G:/sail/installed/x64-windows/debug/lib/libpng16d.lib (found version "1.6.43")
-- Performing Test HAVE_APNG
-- Performing Test HAVE_APNG - Failed
CMake Error at src/sail-codecs/svg/CMakeLists.txt:66 (message):
  SVG: Missing dependencies


-- Configuring incomplete, errors occurred!
```

--- a/ports/sail/portfile.cmake
+++ b/ports/sail/portfile.cmake
@@ -2,7 +2,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO HappySeaFox/sail
     REF "v${VERSION}"
-    SHA512 60a5923e0b95ecfeeb3f6dbc21f82da322ed34bdfd732501e90f89f5bbbb6907c8dbb79d8da09889df3f5a1e9d440c292d539c984d06c815058852b26658155e
+    SHA512 75d797d3fb36e1712cfdd2f0cc13f9bb20b2fc1fe0546e2046ad4356b608adb0efb9c3d17dd1b3c131e445fd5399705c0598d04ba55b1140d271ba86c6e42744
     HEAD_REF master
     PATCHES
         fix-include-directory.patch

--- a/ports/sail/vcpkg.json
+++ b/ports/sail/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "sail",
-  "version-semver": "0.9.3",
+  "version-semver": "0.9.4",
   "description": "The missing small and fast image decoding library for humans (not for machines)",
   "homepage": "https://github.com/HappySeaFox/sail",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7829,7 +7829,7 @@
       "port-version": 0
     },
     "sail": {
-      "baseline": "0.9.3",
+      "baseline": "0.9.4",
       "port-version": 0
     },
     "sajson": {

--- a/versions/s-/sail.json
+++ b/versions/s-/sail.json
@@ -6,11 +6,6 @@
       "port-version": 0
     },
     {
-      "git-tree": "f9e8507b94002b494c4b841e62cd79c81da190d7",
-      "version-semver": "0.9.3",
-      "port-version": 0
-    },
-    {
       "git-tree": "b5fe68901010efcf15a6025a105333d15bb284bf",
       "version-semver": "0.9.1",
       "port-version": 1

--- a/versions/s-/sail.json
+++ b/versions/s-/sail.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "b94b1a44c9796140c96d60203de76f54dd662d7c",
+      "version-semver": "0.9.4",
+      "port-version": 0
+    },
+    {
       "git-tree": "f9e8507b94002b494c4b841e62cd79c81da190d7",
       "version-semver": "0.9.3",
       "port-version": 0

